ui-tests/tst/software/aws/toolkits/jetbrains/uitests/projectwizard/SamTemplateProjectWizardTest.kt

```diff
@@ -0,0 +1,93 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.uitests.projectwizard
+
+import com.intellij.remoterobot.stepsProcessing.log
+import com.intellij.remoterobot.stepsProcessing.step
+import org.junit.BeforeClass
+import org.junit.ClassRule
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import software.aws.toolkits.jetbrains.uitests.fixtures.editor
+import software.aws.toolkits.jetbrains.uitests.fixtures.idea
+import software.aws.toolkits.jetbrains.uitests.fixtures.newProjectWizard
+import software.aws.toolkits.jetbrains.uitests.fixtures.preferencesDialog
+import software.aws.toolkits.jetbrains.uitests.fixtures.welcomeFrame
+import software.aws.toolkits.jetbrains.uitests.rules.Ide
+import software.aws.toolkits.jetbrains.uitests.rules.uiTest
+
+class SamTemplateProjectWizardTest {
+    companion object {
+        @JvmField
+        @ClassRule
+        val ide = Ide(":jetbrains-core:runIdeForUiTests")
```

Comment:
Is Ide meant to be used outside of these tests? should we have a `UiTestIde` that hides this gradle task name?